cms/envs/common.py

```diff
@@ -1238,6 +1239,35 @@
 RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS = 5
 RETRY_ACTIVATION_EMAIL_TIMEOUT = 0.5
 
+STUDIO_SOCK_LINKS = [
+    {
+        'href': 'http://docs.edx.org',
+        'sr_mouseover_text': _('Access documentation on http://docs.edx.org'),
+        'text': _('edX Documentation'),
+        'condition': True
+    }, {
+        'href': 'https://open.edx.org',
+        'sr_mouseover_text': _('Access the Open edX Portal'),
+        'text': _('Open edX Portal'),
+        'condition': True
+    }, {
+        'href': 'https://www.edx.org/course/overview-creating-edx-course-edx-edx101#.VO4eaLPF-n1',
+        'sr_mouseover_text': _('Enroll in edX101: Overview of Creating an edX Course'),
+        'text': _('Enroll in edX101'),
+        'condition': True
+    }, {
+        'href': 'https://www.edx.org/course/creating-course-edx-studio-edx-studiox',
+        'sr_mouseover_text': _('Enroll in StudioX: Creating a Course with edX Studio'),
+        'text': _('Enroll in StudioX'),
+        'condition': True
+    #    }, {
+    #        'href': 'mailto:{email}'.format(email=PARTNER_SUPPORT_EMAIL),
```

Comment:
Maybe try uncommenting this block,
starting by removing the variables/formatting.
Make sure that passes, then add the variables line by line.
With each change, run the test suite locally.